synapse/lib/msgpack.py

```diff
@@ -223,6 +221,10 @@ def feed(self, byts):
             except msgpack.exceptions.OutOfData:
                 break
 
+            except UnicodeDecodeError as exc:
```

Comment:
Are we going to need special load methods for 2->3 conversion due to invalid data at rest?

Reply:
Yeah we will. I went back and forth on whether we should allow loading invalid data, can def discuss.

Reply:
Adjusted to use `replace` by default